src/main/java/org/broadinstitute/hellbender/tools/spark/sv/InsertSizeDistribution.java

```diff
@@ -0,0 +1,157 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.google.api.client.repackaged.com.google.common.annotations.VisibleForTesting;
+import org.apache.commons.math3.distribution.AbstractRealDistribution;
+import org.apache.commons.math3.distribution.LogNormalDistribution;
+import org.apache.commons.math3.distribution.NormalDistribution;
+import org.apache.commons.math3.distribution.RealDistribution;
+import org.broadinstitute.hellbender.exceptions.UserException;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Holds the information characterizing and insert size distribution.
+ */
+public class InsertSizeDistribution implements Serializable {
```

Comment:
I thought we had a convention for class layout that put all fields first, then constructors, then other stuff.  I could be wrong.